Include a timeout to cachito subprocess calls (#457)
release-engineering/cachito: README.md

* `cachito_task_log_format` - the log format that Celery displays when a task is executing. This
  defaults to
  `"[%(asctime)s #%(request_id)s %(name)s %(levelname)s %(module)s.%(funcName)s] %(message)s"`.

> * `cachito_subprocess_timeout` - a number (in seconds) to set a timeout to execute a command by
>   `subprocess` module. Default is 3600 seconds. A timeout is always required, and there is no
>   way provided by Cachito to disable. Set a larger number if it is allowed to wait the subprocess
>   execution for a longer time.

Reviewer: ```suggestion
* `cachito_subprocess_timeout` - a number (in seconds) to set a timeout for commands executed by
  the `subprocess` module. Default is 3600 seconds. A timeout is always required, and there is no
  way provided by Cachito to disable it. Set a larger number to give the subprocess execution more
  time.
```
